index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Music Festival</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="main-nav bg-secondary">
+        <div class="container">
+            <ul class="main-nav-ul float-right">
+                <li class="main-nav-i"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="main-nav-i"><i class="fa-brands fa-twitter"></i></li>
+                <li class="main-nav-i">English</li>
+                <li class="main-nav-i">My Page</li>
+                <li class="main-nav-i">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <nav class="second-nav fixed-top-nav" id="first-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-image"></i>
+            <div class="navigation">
+                <img class="navigation-logo" src="assets/musicFestivalLogo.png" width="75px" alt="Festival">
+                <ul class="second-nav-ul float-right">
+                    <li><a class="second-nav-i" href="about.html">About</a></li><br>
+                    <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                    <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                    <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                    <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                    <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="mobile-nav">
+        <span class="cross">x</span>
+        <ul class="mobile-nav-ul">
+            <li><a class="mobile-nav-i" href="about.html">About</a></li><br>
+            <li><a class="mobile-nav-i" href="#program">Program</a></li><br>
+            <li><a class="mobile-nav-i" href="#join">Join</a></li><br>
+            <li><a class="mobile-nav-i" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="mobile-nav-i" href="#updates">News</a></li><br>
+            <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+        </ul>
+    </nav>
+    <main class="top-section">
```

Comment:
- [ ] As suggested by the previous reviewer, there are some issues in both of your neavbars.kindly consider fixing them. because we need to take account of the margins, paddings, spacing, fonts and colors. these things must be identical to the original deisgn.
   - your top nav background color is not similar to the original design.try applying this color `#3e3e3e`
   - The bottom navbar elements are not positioned rightly. try applying flex and align-items: center in the `first-header` class. to make them look good.
   - The border color of cc campaign is not right. consider making it identical to the original design.
your design
![q](https://github.com/Kgomotso196/FirstCapstoneProject/assets/96683843/bf4fbb75-b411-4a7e-a379-588ed1391cac)

original design
![w](https://github.com/Kgomotso196/FirstCapstoneProject/assets/96683843/6994b7b9-2724-4c1b-a151-c3fc9924f70d)